frontends/article/article-page.tsx

```diff
@@ -1,5 +1,5 @@
 import EditorJS from '@editorjs/editorjs';
-import { Divider, Editor } from 'components';
+import { Author, Divider, Editor } from 'components';
```

Comment:
dublicate import from 'components' folder
you have NOT removed Author component import